source/concepts/harvesting.rst

```diff
@@ -56,24 +70,27 @@ Local harvesting is secure as long as no one accesses your node instance, which
 Delegated harvesting
 ********************
 
-An eligible account may also delegate its importance score to a :ref:`remote node <delegated-harvesting>` for harvesting.
+Delegated harvesting allows using the importance score of an account to create new blocks and receive rewards without having to run a node locally.
 
-Delegated harvesting enables an account to use a **proxy private key** that can be shared with a node securely.
-In other words, you can use the importance score of your account to create new blocks without running a node.
+An eligible account can delegate its importance score to a brand new **proxy account**
+The private key of the proxy account can be shared securely with any node that is running local harvesting thatdelegated harvesting connections.
 
 .. figure:: ../resources/images/diagrams/delegated-harvesting.png
     :align: center
     :width: 400px
 
     Activating delegated harvesting
 
-To enable delegated harvesting, the account owner has to link its **importance score** to a remote account announcing an :ref:`AccountLinkTransaction <account-link-transaction>`.
+To enable delegated harvesting, the account owner has to link its importance score to the proxy account by announcing an :ref:`AccountKeyLinkTransaction <account-key-link-transaction>`.
+In order for the remote account to be accepted for delegated harvesting, the remote account should not have sent nor received any transaction or be linked to another account previously.
+
+.. note:: The remote account cannot be involved in any transaction while it is a delegated account.
 
 Then, the account needs to send a `special encrypted message <https://github.com/nemtech/NIP/blob/master/NIPs/nip-0009.md>`_ to the node via a :doc:`TransferTransaction <transfer-transaction>`.
-The message must contain the remote's account **proxy private key**  encrypted using AES, so that only the recipient will be able to decipher it.
+The message must contain the remote's account **proxy private key**  encrypted using AES, making the transaction only readable by the recipient.
```

Comment:
The message must contain the remote's account **proxy private key**  encrypted using AES, making the transaction readable only by the recipient.